packages/kakaswap-contracts/.env.example

```diff
@@ -2,6 +2,6 @@ GOERLI_RPC_URL=
 CONSENSYS_ZK_GOERLI_RPC_URL=
 POLYGON_ZK_GOERLI_RPC_URL=
 ETHERSCAN_API_KEY=
-# Account 0 in anvil
+# Public address derived from private key 
```

Comment:
can you put the same PRIVATE_KEY as in the kakarot main repot?

```
0x7f109a9e3b0d8ecfba9cc23a3614433ce0fa7ddcc80f2a8f10b222179a5a80d6
```

Reply:
or maybe we should just use the same default mnemonic everywhere, the one in anvil and all devnet, "test test ... test"